elasticsearch/pom.xml

```diff
@@ -29,10 +29,15 @@ LICENSE file.
     <name>ElasticSearch Binding</name>
     <packaging>jar</packaging>
     <properties>
-        <elasticsearch-version>0.19.8</elasticsearch-version>
+        <elasticsearch-version>2.1.1</elasticsearch-version>
     </properties>
     <dependencies>
         <dependency>
+            <groupId>net.java.dev.jna</groupId>
+            <artifactId>jna</artifactId>
+            <version>4.1.0</version>
```

Comment:
Yes, elasticsearch supports jna, though it does have a fallback in case jna isn't supported. 
I don't have a strong opinion on whether to keep it or not, so if you have a good reason to remove it I'm fine with that.